src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -66,59 +66,59 @@ export const evidenceTypeHelp = (
 
 export const disabilityNameTitle = ({ formData }) => {
   return (
-    <legend className="schemaform-block-title schemaform-title-underline">{formData.disability.diagnosticText}</legend>
+    <legend className="schemaform-block-title schemaform-title-underline">{formData.name}</legend>
   );
 };
 
 
 export const facilityDescription = ({ formData }) => {
   return (
-    <p>Tell us about facilities where VA treated you for {formData.disability.diagnosticText}, <strong>after you got your disability rating</strong>.</p>
+    <p>Tell us about facilities where VA treated you for {formData.name}, <strong>after you got your disability rating</strong>.</p>
   );
 };
 
 
 export const treatmentView = ({ formData }) => {
-  const { startTreatment, endTreatment, treatmentCenterName } = formData.treatment;
-  let treatmentPeriod = '';
+  const { from, to } = formData.treatmentDateRange;
 
-  if (startTreatment && endTreatment) {
-    treatmentPeriod = `${startTreatment} — ${endTreatment}`;
-  } else if (startTreatment || endTreatment) {
-    treatmentPeriod = `${(startTreatment || endTreatment)}`;
+  let treatmentPeriod = '';
+  if (from && to) {
+    treatmentPeriod = `${from} — ${to}`;
+  } else if (from || to) {
+    treatmentPeriod = `${(from || to)}`;
```

Comment:
Are from and two both optional? OR does the date range validation now require both? If the later we can simplify this and remove the conditional logic.

Reply:
Yeah it's actually 'from' that's required (but not 'to'). I think it's more clear (if slightly overkill) to leave it as-is so that someone down the line doesn't have to wonder about why we didn't handle the 'from is missing' case. I just updated this requirement in the schema.

Reply:
Thanks for the context @markgreenburg, very helpful.